content/installation/netcore/install/Supported-technology-netcore.md

```diff
@@ -25,6 +25,70 @@ The Contrast .NET Core agent supports analysis of web applications built on the
 > * .NET Core >= 3.0 is currently not supported.
 > * ASP.NET Core applications running under the .NET framework aren't supported.
 
-<!-- ## .NET Core for Azure App Service 
+<!-- ## .NET Core Rule support.  This could be in its own section -->
 
-TBD if we will support in the first release. -->
\ No newline at end of file
+## .NET Core Agent Rule Support
```

Comment:
@DanFiedler I added this section for currently supported and upcoming.  Is this the best place to put or should it go in its own page?